src/njoy21/input/PLOTR/test/PLOTR.test.cpp

```diff
@@ -0,0 +1,779 @@
+      REQUIRE( 1 == plotr.curves[0].second.card9.ithick.value );
+      REQUIRE( 0 == plotr.curves[0].second.card9.ishade.value );
+    }
+//std::string sCard12_3( " 0 /\n" );
+//std::string sCard13_3_1( " 1.0 2.0 1.5 2.5 0.5 1.5 /\n" );
+//std::string sCard13_3_2( " 5.0 5.0 0 0 0 0 /\n" );
+//std::string sCard13_3_3( " 20.1 21.2 10.0 30.0 10.0 25.0 /\n" );
+//std::string sCard13_3_4( " /\n" );
+    THEN( "the Card12 input values can be verified: 0" ){
+      REQUIRE( 0 == plotr.curves[0].second.freeform->card12.nform.value );
+    }
+
+    THEN( "the Card13 input values can be verified" ){
+      REQUIRE( Approx( 1.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).xdata.value ) );
+      REQUIRE( Approx( 2.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).ydata.value ) );
+      REQUIRE( Approx( 1.5 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).yerr1.value ) );
+      REQUIRE( Approx( 2.5 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).yerr2.value ) );
+      REQUIRE( Approx( 0.5 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).xerr1.value ) );
+      REQUIRE( Approx( 1.5 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(0).xerr2.value ) );
+
+      REQUIRE( Approx( 5.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).xdata.value ) );
+      REQUIRE( Approx( 5.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).ydata.value ) );
+      REQUIRE( Approx( 0.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).yerr1.value ) );
+      REQUIRE( Approx( 0.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).yerr2.value ) );
+      REQUIRE( Approx( 0.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).xerr1.value ) );
+      REQUIRE( Approx( 0.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(1).xerr2.value ) );
+
+      REQUIRE( Approx( 20.1 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).xdata.value ) );
+      REQUIRE( Approx( 21.2 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).ydata.value ) );
+      REQUIRE( Approx( 10.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).yerr1.value ) );
+      REQUIRE( Approx( 30.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).yerr2.value ) );
+      REQUIRE( Approx( 10.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).xerr1.value ) );
+      REQUIRE( Approx( 25.0 ) ==
+            *( plotr.curves[0].second.freeform->card13->at(2).xerr2.value ) );
+    }
+
+    THEN( "the remaining cards are empty" ){
+      REQUIRE( std::nullopt == plotr.curves[0].second.plot->card7 );
+      REQUIRE( std::nullopt == plotr.curves[0].second.card10 );
+      REQUIRE( std::nullopt == plotr.curves[0].second.card10a );
+      REQUIRE( std::nullopt == plotr.curves[0].second.card11 );
+    }
+  }//WHEN
```

Comment:
Very nicely done here. This is a complicated test that needs to be made and you've done a good job. One thing to add is we need to add a test where there is more than one curve plotted. They can both be exactly the same data, but we need to test more than one.